charms/kserve-controller/requirements-integration.in

```diff
@@ -1,6 +1,9 @@
 aiohttp
 jinja2
-juju==2.9.44
+# Pinning to <4.0 due to compatibility with the 3.1 controller version
+juju<4.0
+lightkube
+lightkube-models
```

Comment:
I believe models will be installed as a dependency of lightkube, no need to list it here.

```suggestion

```